selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1296,32 +1330,64 @@ public void openJavaDocPopUp() {
    * @param text is the text of elements active line
    */
   public void waitTextElementsActiveLine(final String text) {
-    waitActiveEditor();
+    waitActive();
     redrawDriverWait.until(visibilityOf(activeLineXpath));
     redrawDriverWait.until(
         (ExpectedCondition<Boolean>) driver -> activeLineXpath.getText().contains(text));
   }
 
   /** get positions of the current line and char */
-  public Pair<Integer, Integer> getCurrentCursorPositions() {
-    waitActiveEditor();
+  public Pair<Integer, Integer> getCurrentCursorPositionsFromVisible() {
+    waitActive();
     WebElement currentActiveElement =
         activeLineNumbers.stream().filter(webElement -> webElement.isDisplayed()).findFirst().get();
-    int[] currentCursorPosition =
-        Arrays.asList(
-                redrawDriverWait.until(visibilityOf(currentActiveElement)).getText().split(":"))
+    return getCursorPositionFromWebElement(currentActiveElement);
+  }
+
+  /**
+   * Check that editor active and focused on first step. After that get cursor position. Note! For
+   * correct work a editor must be active.
+   *
+   * @return char and line position from current visible, active and focused editor (usual uses with
+   *     split editor)
+   */
+  public Pair<Integer, Integer> getCursorPositionsFromCurentActive() {
```

Comment:
1. If active editor can't be non-current, I would simplify the name to **getCursorPositionsFromActive()**.
1. There is a typo "_curent_" > "_current_".